source/content/guides/composer-convert.md

```diff
@@ -161,11 +176,12 @@ From the `$site-composer` directory, run the following:
 
 ```bash
 composer prepare-for-pantheon
+composer install --no-dev
 ```
 
-The output `> DrupalProject\composer\ScriptHandler::prepareForPantheon` means the command was successful. This should modify the `.gitignore` file and cleanup any errant `.git` directories in the codebase, to prepare your new code for direct deployment to Pantheon.
+This should modify the `.gitignore` file and cleanup any errant `.git` directories in the codebase, to prepare your new code for direct deployment to Pantheon.
 
-### Commit
+## Commit
```

Comment:
While I agree that an H2 makes more sense in context of the other headers, I'd like to see if we can't collapse this ToC overall to use H3s more effectively.

Reply:
this makes sense